shell/platform/android/platform_view_android_jni.h

```diff
@@ -34,6 +34,8 @@ void FlutterViewUpdateCustomAccessibilityActions(JNIEnv* env,
 
 void FlutterViewOnFirstFrame(JNIEnv* env, jobject obj);
 
+void FlutterViewOnStarted(JNIEnv* env, jobject obj, jboolean success);
```

Comment:
"started" is a rather nebulous term. Can be make this more descriptive like `OnRootIsolateDidStart` or something?